compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -933,6 +933,12 @@ error.invalid.interface.of.non.abstract.object=\
 error.unimplemented.referenced.method.in.class=\
   no implementation found for the method ''{0}'' of class ''{1}''
 
+error.unimplemented.referenced.method.in.service.declaration=\
+  no implementation found for the method ''{0}'' of service declaration ''{1}''
```

Comment:
Isn't this a bit misleading? The error is that the service decl hasn't implemented the method in the type descriptor it specifies. Judging by the error message in the test case for this, the type descriptor is passed to `{{1}}`. How about something like 
```
implementation not found for the method 'exec' of type 'ServType' referred in the service declaration
```
or something similar. Something that makes it clear that the issue is that the service decl has referred a type, but it hasn't provided an implementation for a method described in the type.

Reply:
Or
```
service declaration expected to provide an implementation for the method 'exec' of type 'ServType'
```

Reply:
No, the `type` of the type that is being implemented is passed as the second arg to error message.